lua/telescope/builtin/files.lua

```diff
@@ -230,17 +237,27 @@ files.file_browser = function(opts)
 
   pickers.new(opts, {
     prompt_title = 'Find Files',
-    finder = gen_new_finder(opts.cwd),
+    finder = gen_new_finder({ path = opts.cwd, hidden = opts.hidden }),
```

Comment:
we should pass user's hidden opt here because if I do `opts.hidden or new_opts.hidden` then the hidden toggler stopped working